sympy/matrices/common.py

```diff
@@ -1888,6 +1888,7 @@ def subs(self, *args, **kwargs):  # should mirror core.basic.subs
         """
         return self.applyfunc(lambda x: x.subs(*args, **kwargs))
 
+
```

Comment:
Only a blank line addition in this file.
Can you remove it?

Reply:
yes .